californium-core/src/test/java/org/eclipse/californium/core/test/ObserveTest.java

```diff
@@ -100,8 +104,7 @@
 	private String uriX;
 	private String uriY;
 
-	private int notificationCounter = 0;
-	private int resetCounter = 0;
+	private AtomicInteger resetCounter = new AtomicInteger(0);
```

Comment:
Its used in the inner class `ServerMessageInterceptor` and to move in to method level would require to change this also.